analysers/experiments/cpu.py

```diff
@@ -0,0 +1,87 @@
+import sys
+import json
+import io
+import gzip
+import uuid
+import math
+
+from pyspark_cassandra import CassandraSparkContext
+from pyspark_cassandra import RowFormat
+from pyspark import SparkConf
+
+# Takes arguments: Spark master, Cassandra host, Minio host, path of the files
+sparkMaster = sys.argv[1]
+cassandraHost = sys.argv[2]
+trialID = sys.argv[3]
+experimentID = trialID.split("_")[0]
+cassandraKeyspace = "benchflow"
```

Comment:
`cassandraKeyspace`, `srcTable`, `destTable` should be defined as part of the configuration file and passed as parameters